### ISSUE TITLE
Modal web component: Documentation and Sketch update

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/va.gov-team#35272]
## Description
Dependent on #35255 and #35260

Two things need to happen as part of this ticket:
1. We need to update design.va.gov to point to the new web component version of the modal and update the documentation there if anything was changed in the conversion to a web component that needs to be noted.
2. We need to check in with @GnatalieH to make sure that any changes that were made are added to Sketch if necessary.

## Acceptance Criteria
- [ ] Update design.va.gov
- [ ] Check with Natalie on Sketch updates

**Comment by GnatalieH** [comment]
Here's a link to a ticket I created last month for [updating the Modal documentation](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/626). Should I close out my older ticket? Martha had requested usage and accessibility guidelines, and Allison is requesting max length for modal content specifications. Thx.

**Comment by k80bowman** [comment]
Let's keep them separate for now. The other ticket sounds like it will be a more extensive effort on your part, whereas this ticket is more about updating design.va.gov to point to the new modal web component instead of the React component. I am updating this ticket, however, to include checking in with you to make sure the Sketch library is up to date with any changes that may have been made to this component.

**Comment by GnatalieH** [comment]
I think this is a duplicate of #39968.

**Comment by k80bowman** [comment]
They are not quite duplicates, but they definitely overlap. It does look like all the work for this ticket is covered by #39968, so we can close this one. Thanks.
